### PR TITLE
Update Drone Scheduler

### PR DESCRIPTION
**Pull request by v-fearam**
Net8
Library versions
KeyVault deprecated library changed

### CODE REVIEW

File: src/shipping/dronescheduler/Dockerfile [hunk 1]
```diff
@@ -19,33 +17,17 @@ COPY dronescheduler/Fabrikam.DroneDelivery.DroneSchedulerService/. ./droneschedu
...
-RUN \
-    # Ensures the entry point is executable
-    chmod ugo+x /app/run.sh
-
-RUN chown -R $user.$user /app
-
-# Set it for subsequent commands
-USER $user
+RUN ls -la
+ENTRYPOINT ["dotnet", "Fabrikam.DroneDelivery.DroneSchedulerService.dll"]
```
Comment (ferantivero) [inline src/shipping/dronescheduler/Dockerfile:32]: Note: we should investigate a bit more before changing this best practice for containers.

File: src/shipping/dronescheduler/scripts/run.sh [hunk 2]
```diff
@@ -1,2 +0,0 @@
-#!/bin/bash
```
Comment (ferantivero) [inline src/shipping/dronescheduler/scripts/run.sh:1]: Please let's do not delete this for now.

**Review by ferantivero — CHANGES_REQUESTED** [review]

File: src/shipping/dronescheduler/Dockerfile (hunk 1 above)
Comment (v-fearam) [reply src/shipping/dronescheduler/Dockerfile:32]: It was not working and I couldn't make it work again. Looking forward to working together on it.

**Review by ferantivero — APPROVED** [review]
LGTM

**Pull request merged**